### PR TITLE
add ManageDocuments scopeName

### PR DESCRIPTION
**Pull request by abrantesarthur**
## Related Issues

- _[none]_

## Security Implications

_[none]_

## System Availability

_[none]_

### CODE REVIEW

**Comment by abrantesarthur** [comment]
I'm not sure under which `TranscendProduct` I was supposed to add the scope, so I created a new one. Let me know if that was a mistake.

**Review by giacaglia — COMMENTED** [review]
will wait for @michaelfarrell76's approval

File: src/scopes.ts
Comment (giacaglia) [inline src/scopes.ts:489]: I think it should be either Admin or Assessments. Will wait for Mike's feedback here

File: src/scopes.ts
Comment (michaelfarrell76) [inline src/scopes.ts:54]:
```suggestion
  ManageAssessments = 'manageAssessments',
  ViewAssessments = 'viewAssessments',
```

File: src/scopes.ts
Comment (michaelfarrell76) [reply src/scopes.ts:489]: i think this is right to make the new product, but lets also tag it with `TranscendProduct.DataMapping` as this will be included in that offering for existing customers for the time being

File: src/scopes.ts
Comment (michaelfarrell76) [inline src/scopes.ts:485]:
```suggestion
    dependencies: [ScopeName.ViewAssessments],
```

**Review by michaelfarrell76 — COMMENTED** [review]
few edits, thanks!!

File: src/scopes.ts
Comment (abrantesarthur) [reply src/scopes.ts:489]: Sounds good.

Comment (abrantesarthur) [reply src/scopes.ts]: done

File: src/scopes.ts
Comment (abrantesarthur) [reply src/scopes.ts]: done

**Comment by michaelfarrell76** [comment]
youll also need to bump the package.json version

File: src/scopes.ts
Comment (abrantesarthur) [reply src/scopes.ts:485]: done

File: src/scopes.ts
Comment (michaelfarrell76) [inline src/scopes.ts:277]: typo

**Comment by michaelfarrell76** [comment]
missing the definition for ViewAssessments as well

File: src/scopes.ts
Comment (abrantesarthur) [reply src/scopes.ts]: fixed

**Comment by abrantesarthur** [comment]
> missing the definition for ViewAssessments as well

Don't I just need to add it to ScopeName?

File: src/scopes.ts
Comment (michaelfarrell76) [inline src/scopes.ts:485]: @abrantesarthur you need something like this

Comment (abrantesarthur) [reply src/scopes.ts:492]: true

File: src/scopes.ts
Comment (michaelfarrell76) [inline src/scopes.ts:495]:
```suggestion
    description: 'Manage and edit assessments and assessment templates',
```

File: src/scopes.ts
Comment (michaelfarrell76) [inline src/scopes.ts:488]:
```suggestion
    description: 'View the assessments and assessment templates.',
```

File: src/scopes.ts
Comment (michaelfarrell76) [inline src/scopes.ts:55]:
```suggestion
  ManageAssessments = 'manageAssessments',
  ApproveAssessments = 'approveAssessments',
```

@abrantesarthur apologies, i forgot about this, but can we add one more scope? to allow for different people to be approving these vs different people editing them?

Comment (abrantesarthur) [reply src/scopes.ts:55]: No problem

File: src/scopes.ts
Comment (giacaglia) [inline src/scopes.ts:499]: Need to add here approveAssessments as well

Comment (abrantesarthur) [reply src/scopes.ts:499]: done

**Review by michaelfarrell76 — APPROVED** [review]
<33333

**Comment by michaelfarrell76** [comment]
gonna merge to pull into my other pr

**Pull request merged**
